internal/store/persistentvolume.go

```diff
@@ -32,11 +32,45 @@ import (
 )
 
 var (
+	descPersistentVolumeClaimRefName          = "kube_persistentvolume_claim_ref_info"
```

Comment:
We tend to reserve the `_info` suffix for things that never change over the lifetime of an object, so this should just be `kube_persistentvolume_claim_ref`.